App/Components/_colors.js

```diff
@@ -0,0 +1,13 @@
+const colors = {
```

Comment:
omg, these resources look great. I'm practically committing every mistake they say not to commit lol. 

I know we also briefly said we might have to re-think the colors we're using for this app to align more closely with colors related to HIV (e.g., shades of red and pink), so I think I'm going to hold off on renaming the colors for now. But good thing to keep in mind for later on definitely!